.clang-format

```diff
@@ -1,26 +1,30 @@
 ---
 Language:        Cpp
-BasedOnStyle:  LLVM
 AccessModifierOffset: -4
 AlignAfterOpenBracket: Align
+AlignConsecutiveMacros: false
 AlignConsecutiveAssignments: false
 AlignConsecutiveDeclarations: false
 AlignEscapedNewlines: Left
-AlignOperands: true
+AlignOperands:   true
```

Comment:
easter egg adding a wrongly formatted formatter config file? :-D

Reply:
Nope, that is what `clang-format-9 --dump-config` outputs. Don't want to touch that for easier updates

Reply:
strange enough